extensions/amp-fx-collection/0.1/providers/amp-fx-presets.js

```diff
@@ -109,4 +109,70 @@ export const Presets = {
       });
     },
   },
+  'fade-in-scroll': {
+    isFxTypeSupported(win) {
+      user().assert(isExperimentOn(win, 'amp-fx-fade-in-scroll'),
```

Comment:
This is a holdover and doesn't matter for this PR, but I feel like isFxTypeSupported might be redundant given the existance userAsserts?